Core/Tests/Speckle.Core.Tests.Unit/Api/Operations/OperationsReceiveTests.Exceptional.cs

```diff
@@ -33,7 +33,7 @@ public void Receive_OperationCanceled_ExceptionThrown(string id)
     ctc.Cancel();
 
     MemoryTransport emptyTransport2 = new();
-    Assert.ThrowsAsync<OperationCanceledException>(async () =>
+    Assert.CatchAsync<OperationCanceledException>(async () =>
```

Comment:
Is `CatchAsync` the recommended pattern to assert an exception type here? Asking out of ignorance

Reply:
ThrowsAsync tests if the method throws the EXACT type
CatchAsync allows subtypes

In this case, the spirit of the test was to make sure it an `OperationCancelledException`, we don't care if its a subtype like `TaskCancelledException`, so I after learning of the existence of `CatchAsync`, I decided to update the functions where we would be ok with subtypes.